src/test/kotlin/se/uulm/snowballr/backend/repository/InvitationTokenTableRepoTest.kt

```diff
@@ -118,6 +120,53 @@ class InvitationTokenTableRepoTest : RepositoryTest(arrayOf(UserTable, ProjectTa
             }
     }
 
+    @Nested
+    inner class GetActiveInvitationTokenForProject {
+        private fun getRandomTestEmail() = "${UUID.randomUUID().toString().substring(0, 5)}@example.com"
+
+        private suspend fun insertActiveTestToken(projectId: UUID, token: String) =
+            insertTestToken(getRandomTestEmail(), projectId, token)
+
+        private suspend fun insertInactiveTestToken(projectId: UUID) = insertTestToken(
+            getRandomTestEmail(),
+            projectId,
+            "an-inactive-token",
+            expiresAt = OffsetDateTime.now().minusDays(1),
+        )
+
+        @Test
+        fun `When active tokens exist, then they are returned`() = runTest {
+            val projectId = insertProjectAndGetId(createdBy = testUserId)
+            val projectId2 = insertProjectAndGetId(createdBy = testUserId)
+
+            insertActiveTestToken(projectId, "an-active-token")
+            insertActiveTestToken(projectId2, "token-in-another-project")
+            insertInactiveTestToken(projectId)
+
+            val activeInvitationTokenInProject = repo.getActiveInvitationTokenForProject(projectId)
+            assertEquals(1, activeInvitationTokenInProject.size)
```

Comment:
```suggestion
            assertThat(activeInvitationTokenInProject).hasSize(1)
```

Just an idea, to make it semantically more accurate, but equals is fine as well.